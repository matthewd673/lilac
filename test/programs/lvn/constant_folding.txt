func main() -> void
  i32 $b = 10i32
  i32 $c = $b
  i32 $a = $b + $c
  i32 $d = $a - $c
  i32 $e = $d << 1i32
end
