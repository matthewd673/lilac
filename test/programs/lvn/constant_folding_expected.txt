i32 b = 10i32
i32 c = 10i32
i32 a = 20i32
i32 d = 10i32
i32 e = 20i32
